src/bci_build/package/kiwi/README.md.j2

```diff
@@ -0,0 +1,32 @@
+# {{ image.pretty_name }} container image
+
+{% include 'badges.j2' %}
+
+## Description
+
+[KIWI](https://osinside.github.io/kiwi/) is a command-line utility for building
+Linux system appliances. An appliance is a ready to use image of an operating
+system that includes a pre-configured application for a specific use case. The
+appliance is provided as an image file and needs to be deployed to or
+activated in the target system or service.
+
+KIWI NG can create different types of appliances. In addition to the standard installation ISOs and images for virtual machines, KIWI NG can also build images that boot via PXE or Vagrant boxes.
```

Comment:
nit: is it KIWI or KIWI NG? I think you should stick with one form in the entire readme. The official docs use KIWI NG everywhere it seems.